<launch>

<node pkg="people_aware_nav" type="move_base_constrained" respawn="false" name="move_base_node" output="screen" machine="three">
  <param name="global_frame" value="map" />
  <param name="robot_base_frame" value="base_link" />

  <param name="inscribed_radius" value="0.325" />
  <param name="circumscribed_radius" value="0.46" />
  <param name="inflation_radius" value="0.55" />

  <rosparam file="$(find people_aware_nav)/launch/costmap_common_params.yaml" command="load" ns="navfn" />
  <rosparam file="$(find people_aware_nav)/launch/costmap_common_params.yaml" command="load" ns="base_local_planner" />
  <rosparam file="$(find 2dnav_pr2)/move_base/navfn_params.yaml" command="load" />
  <rosparam file="$(find 2dnav_pr2)/move_base/base_local_planner_params.yaml" command="load" />
</node>
</launch>
